qiskit/providers/ibmq/job/ibmqjob.py

```diff
@@ -476,7 +479,14 @@ def time_per_step(self) -> Optional[Dict]:
         """
         if not self._time_per_step or self._status not in JOB_FINAL_STATES:
             self.refresh()
-        return self._time_per_step
+        time_per_step_local = {}
+        if self._time_per_step:
+            for step_name, time_data in self._time_per_step.items():
+                time_data_utc_dt = str_to_datetime(time_data)
```

Comment:
Doesn't `utc_to_local()` take a str type?

Reply:
Yeah, it does. I was using `str_to_datetime` since the string was in a specific format. But, as mentioned `dateutil.parser` might be better, which `utc_to_local` uses. So, I'll use `utc_to_local` 👍